### ISSUE TITLE
Which is the entry point of synthea? App.java?

### ISSUE THREAD

**Issue by meSmashsta** [synthetichealth/synthea#352]
There are multiple ways to run synthea. The first one is the command line by executing the 'run_synthea' file, the second one is going to the App.java which contains the entry point (main method).

I saw that 'run_synthea' runs gradlew, but I don't know which class is their entry point. Does the 'run_synthea' also uses the App.java as the starting point?

**Comment by dehall** [comment]
Yes - App.java is the primary entry point that gets called when using `run_synthea`. 

Re: the difference in number of patients, are you passing command line arguments to the `run_synthea` command? By default both will use the `generate.default_population` setting in `src/main/resources/synthea.properties` (which is initially set to 1), but you can override that in the file or with the `-p` command-line argument.
